packages/react-api/src/hooks/gpu-filter-utils.js

```diff
@@ -0,0 +1,88 @@
+import { _buildFeatureFilter, _FilterTypes } from '@carto/react-core/';
+
+// Don't change this value to maintain compatibility with builder
+export const MAX_GPU_FILTERS = 4;
```

Comment:
I think this is probably a limitation also from deck.gl side, a max number of 4.

Reply:
Yeap